workers/unity/Packages/com.improbable.gdk.core/Systems/ReactiveComponentSendSystem.cs

```diff
@@ -106,8 +109,8 @@ private void PopulateDefaultComponentReplicators()
         private struct ComponentReplicator
         {
             public uint ComponentId;
-            public ComponentReplicationHandler Handler;
-            public ComponentGroup UpdateGroup;
+            public IReactiveComponentReplicationHandler Handler;
+            public ComponentGroup EventGroup;
```

Comment:
Are events and commands still using reactive components?
if yes, are there plans to move those to non-reactive components as well?

Reply:
The old reactive components are all still there if you enable them. What are non reactive components? Do you mean just have potentially empty static lists on things? If so that can be added as a feature module on top of the new stuff added in.

Reply:
nvm that, I think I was slightly confused back then. If I understand it correctly, we now have two ways to process commands, right? One is via the command managers and the other one is via reactive components.

One question that I have regarding that: How would that work in an ECS system? I assume if we don't want to use reactive components, we would need to get a hold of the correct managers and query them all time to check whether we received a request / response

Reply:
Basically yes. It's not the ideal API but it's meant to be a low level API that we make other things from.